packages/cli-repl/src/mongosh-repl.spec.ts

```diff
@@ -643,5 +643,36 @@ describe('MongoshNodeRepl', () => {
       expect(output).to.contain('> ');
       expect(output).to.not.contain('error');
     });
+
+    it('changes the prompt when db is reassigned', async() => {
+      const connectionInfo = {
+        extraInfo: {
+          uri: 'mongodb://localhost:27017/test',
+          is_localhost: true
+        },
+        buildInfo: {
+          version: '4.4.1',
+          modules: ['enterprise']
+        }
+      };
+
+      sp.getConnectionInfo.resolves(connectionInfo);
+      sp.getNewConnection.callsFake(async() => {
+        Object.assign(connectionInfo.extraInfo, {
+          is_localhost: true,
+          is_data_lake: true
+        });
+        return sp;
+      });
+      sp.platform = 2; // ReplPlatform.CLI ... let's maybe stop using an enum for this
```

Comment:
We can make it a string enum with some human readable values 😄

Reply:
Yeah … although I’d personally lean towards using  something`'cli' | 'java-shell' | 'compass' | 'browser'` – I think this is one of those things @rose-m might have meant yesterday?